client/app/components/query-snippets/QuerySnippetDialog.jsx

```diff
@@ -79,6 +78,7 @@ class QuerySnippetDialog extends React.Component {
           fields={formFields}
           onSubmit={this.handleSubmit}
           hideSubmitButton
+          feedbackIcons
```

Comment:
An alternative that I think makes better sense https://github.com/getredash/redash/pull/3974

Reply:
Should we keep the icons for the Query Snippets context or...
```suggestion
```

Reply:
It's a really nice and fun UX, but not a must. I have no strong opinion.